app/models/seaplane.rb

```diff
@@ -8,4 +8,10 @@ class Seaplane < ApplicationRecord
   validates :price, presence: true, numericality: { only_integer: true }
   validates :category, presence: true #(to be implemented)
   validates :capacity, presence: true, numericality: { only_integer: true }, inclusion: { in: 1..6 }
+
+  def unavailable_dates
```

Comment:
Do we define methods in model? I thought they should only be in controllers but I might be wrong. Just checking